Godeps/Godeps.json

```diff
@@ -58,6 +68,131 @@
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/corehandlers",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/credentials",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/credentials/ec2rolecreds",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/credentials/endpointcreds",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/credentials/stscreds",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/defaults",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/ec2metadata",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/request",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/session",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/aws/signer/v4",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/endpoints",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/ec2query",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/query",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/query/queryutil",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/rest",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/protocol/xml/xmlutil",
+			"Comment": "v1.4.20-8-ged981a1d",
+			"Rev": "ed981a1d5ee78d20547091d0697a711e5185d07f"
+		},
+		{
+			"ImportPath": "github.com/aws/aws-sdk-go/private/waiter",
```

Comment:
we should probably get rid of our dependency on `github.com/hashicorp/consul`, seems like it's adding a lot of bloat for us